packages/p2p-v2/src/pages/my-ads/components/AdFormController/AdFormController.tsx

```diff
@@ -22,7 +22,7 @@ const AdFormController = ({
     return (
         <div className='p2p-v2-ad-form-controller'>
             {getCurrentStep() === 1 && (
-                <Button color='black' onClick={onCancel} variant='outlined'>
+                <Button color='black' onClick={() => onCancel?.()} variant='outlined'>
```

Comment:
shouldn't we require onCancel here?

Reply:
the cancel button is there only for 1st stepper. other steppers have previous button instead.

Reply:
Why would there be a button, if there's no click handler.

`getCurrentStep() === 1` if this is true then there must be an `onCancel`, right?